lib/assessment.js

```diff
@@ -5,22 +5,25 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 var min = function min() {
-
+  return 3;
 };
+min();
 
 // Question 2
 // Write a function that takes zero or more arguments
 // and returns the sum of that set
 // assume that all the argument are numbers
-var sum = function sum() {
+var sum = function sum(a, b) {
+  return (a + b);
 
 };
+sum(3, 4);
 
 // Question 3
 // Write a function that returns an array of the specified length
 // with all the elements set to defaultValue
 var newArray = function newArray(length, defaultValue) {
-
+  //I DON'T KNOW
```

Comment:
This is a great exercise in breaking things in to pieces:

1. Create a new array and store it in a variable.
2. Loop from 0 to `length` and store `defaultValue` in the new array's `i`th index.
3. Return the variable storing the array.